woodwork/table_accessor.py

```diff
@@ -39,11 +39,7 @@
 dd = import_or_none('dask.dataframe')
 ks = import_or_none('databricks.koalas')
 
-from woodwork.typing import (
-    AnyDataFrame,
-    ColumnName,
-    UseStandardTagsDict
-)
+from woodwork.typing import AnyDataFrame, ColumnName, UseStandardTagsDict
```

Comment:
this needs to be moved to be right below `from woodwork.type_sys.utils import _is_numeric_series, col_is_datetime`